extensions/amp-story/1.0/amp-story.js

```diff
@@ -139,6 +139,9 @@ const DESKTOP_WIDTH_THRESHOLD = 1024;
 /** @private @const {number} */
 const DESKTOP_HEIGHT_THRESHOLD = 550;
 
+/** @private @const {string} */
+const DESKTOP_ONE_PANEL_ASPECT_RATIO_THRESHOLD = '3 / 4';
```

Comment:
This could trigger the desktop one panel experience on small screens, which would conflict the min width / min height that you specified in the CSS, right?
Can you document your thinking a bit more in the issue linked on that PR? I'm curious to know more about how you want to build the UI type triggering logic so it works for all devices

Reply:
Ok sorry this makes a lot more sense after seeing your revisions on the sizing PR.

Reply:
It is true that the desktop one panel experience could be triggered on small, square screens (like foldable phones). Or small landscape orientation tablets.
I feel like we're moving towards aspect-ratio driven UI rather than device-type. I wonder if we want to re-consider the UI type naming conventions at some point (not necessarily part of this project).